src/Module.Server/Common/CrpgTeamSelectComponent.cs

```diff
@@ -402,8 +402,11 @@ private bool IsNativeBalancerEnabled()
 
     private void LogRoundResult()
     {
+        MissionScoreboardComponent missionBehavior = Mission.Current.GetMissionBehavior<MissionScoreboardComponent>();

```

Comment:
This component could be injected in the constructor. Alternatively, isn't this info available in the round controller?

Reply:
Also could you give a proper name to the variable